tests/sqlparser_common.rs

```diff
@@ -1953,11 +1976,26 @@ fn parse_derived_tables() {
         }))
     );
 
-    let res = parse_sql_statements("SELECT * FROM ((SELECT 1) AS t)");
```

Comment:
```
// Nesting a subquery in parentheses is non-standard, but supported in Snowflake SQL
```

Please check that it is.

Reply:
Added , thank for the help with the comment wording. 
(checked in snowflake..)